Index: passenger-2.2.11/Rakefile
===================================================================
--- passenger-2.2.11.orig/Rakefile	2010-06-03 00:20:09.000000000 +0300
+++ passenger-2.2.11/Rakefile	2010-06-03 00:23:33.000000000 +0300
@@ -741,11 +741,8 @@
 	include Config
 	fakeroot = "pkg/fakeroot"
 
-	# We don't use CONFIG['archdir'] and the like because we want
-	# the files to be installed to /usr, and the Ruby interpreter
-	# on the packaging machine might be in /usr/local.
-	libdir = "#{fakeroot}/usr/lib/ruby/#{CONFIG['ruby_version']}"
-	extdir = "#{libdir}/#{CONFIG['arch']}"
+	libdir = "#{fakeroot}#{CONFIG['rubylibdir']}"
+	extdir = "#{fakeroot}#{CONFIG['archdir']}"
 	bindir = "#{fakeroot}/usr/bin"
 	docdir = "#{fakeroot}/usr/share/doc/phusion_passenger"
 	libexecdir = "#{fakeroot}/usr/lib/phusion_passenger"
